dns/zone-configs/k8s.io.yaml

```diff
@@ -240,3 +240,7 @@ velodrome:
 kind.sigs:
   type: CNAME
   value: k8s-kind.netlify.com.
+# https://github.com/kubernetes-sigs/release-notes docs (@jeefy)
+relnotes:
```

Comment:
Please keep this list alphabetical.

We generally try to make the same name work as both k8s.io and kubernetes.io, but given that this is external I presume we won't want TLS for both ?

Reply:
Oh this was already added in https://github.com/kubernetes/k8s.io/pull/264/files#diff-72d1f1b649e651304c1e89efa0533680

/close